assets/global.js

```diff
@@ -764,9 +784,17 @@ class SlideshowComponent extends SliderComponent {
     } else if (isLastSlide && event.currentTarget.name === 'next') {
       this.slideScrollPosition = 0;
     }
-    this.slider.scrollTo({
-      left: this.slideScrollPosition,
-    });
+    if (this.querySelector('.announcement-bar-slider')) {
```

Comment:
Can we save `this.querySelector('.announcement-bar-slider')` value to a class field, instead of querying multiple times?